ui/src/components/Timer.tsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from 'react';
+
+const Timer = () => {
+  const [minutes, setMinutes] = useState('00');
+  const [seconds, setSeconds] = useState('00');
```

Comment:
po co trzymać minuty i sekundy osobno? Proponowałbym stworzyć obiekt daty który będzie naszym startem (lub chyba nawet lepiej przyjąć go jako argument) i potem obliczać różnice między nim a obecną datą. Następnie wyświetlać to przy pomocy obiektu [Intl](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/format) jeśli nie chcemy używać żadnej bilbioteki do dat. Chociaż tutaj też proponowałbym użycie np https://date-fns.org/ łatwiej wtedy będzie obliczać róznice w czasie i wyświetlać w określonych formatach